mitmproxy/contrib/kaitaistruct/jpeg.py

```diff
@@ -17,7 +17,7 @@ class Jpeg(KaitaiStruct):
     
     Format is organized as a container format, serving multiple
     "segments", each starting with a magic and a marker. JFIF standard
-    dictates order and mandatory apperance of segments:
+    dictates order and mandatory appearance of segments:
```

Comment:
This is a generated file, please either fix it upstream or not change it here. Thanks!